server/src/galapaRepository.ts

```diff
@@ -138,9 +127,16 @@ export class GalapaRepository {
         island.spots = island.spots.concat(
             this.db.create(Spot, newSpots),
         )
+        await this.db.save(island)
+
         lot.centerSpot = island.getSpotAtCoords(lot.center!)
```

Comment:
The genesis lot problem existed because the center spot was saved twice. This led to all spots being offset by one